tests/e2e/adapters/vertexai/chat.test.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2024 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BaseMessage } from "@/llms/primitives/message.js";
+import { expect } from "vitest";
+import { verifyDeserialization } from "@tests/e2e/utils.js";
+import { VertexAIChatLLM } from "@/adapters/vertexai/chat.js";
+
+describe.runIf(
+  [process.env.GOOGLE_APPLICATION_CREDENTIALS, process.env.GCP_VERTEXAI_PROJECT].every((env) =>
+    Boolean(env),
+  ),
+)("GCP Vertex AI", () => {
```

Comment:
`describe.runIf([getEnv('GOOGLE_APPLICATION_CREDENTIALS', 'GCP_VERTEXAI_PROJECT')].every(Boolean))`

or just extract the env values into standalone variables as long as you are using them in the rest of the code